internal/BUILD.bazel

```diff
@@ -82,12 +89,47 @@ ts_library(
     name = "test_lib",
     srcs = glob(["tsc_wrapped/*_test.ts"]) + ["tsc_wrapped/test_support.ts"],
     tsconfig = "//internal:tsc_wrapped/tsconfig.json",
-    deps = [":tsc_wrapped"],
+    deps = [
+      ":tsc_wrapped",
+      "@npm//:typescript",
+      "@npm//:@types/node",
+      "@npm//:@types/jasmine",
+    ],
 )
 
 jasmine_node_test(
     name = "test",
     srcs = [],
-    deps = [":test_lib"],
-    node_modules = "//:node_modules",
+    deps = [
+      "@npm//:jasmine",
+      "@npm//:typescript",
+      ":test_lib",
+    ],
+)
+
+genrule(
+    name = "license_copy",
+    srcs = ["//:LICENSE"],
+    outs = ["LICENSE"],
+    cmd = "cp $< $@",
+)
+
+genrule(
+    name = "worker_protocol_copy",
+    srcs = ["//third_party/github.com/bazelbuild/bazel/src/main/protobuf:worker_protocol.proto"],
+    outs = ["worker_protocol.proto"],
+    cmd = "cp $< $@",
+)
+
+npm_package(
```

Comment:
too bad this can't be `//internal/tsc_wrapped:npm_package` but I reached the same conclusion as you did